stdlib/public/core/UnicodeGraphemeBreakProperty.swift

```diff
@@ -0,0 +1,92 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftShims
+
+extension Unicode {
+  internal enum GraphemeBreakProperty {
```

Comment:
Let's underscore this. We will probably want to make API one day and that makes it easier.